server/config/searchFields/config.json

```diff
@@ -0,0 +1,811 @@
+{
+  "BF": [
+    {
+      "validation": [],
+      "props": {
```

Comment:
I think we could remove props and also consider renaming label to something like "description" or "title"?

Reply:
I think this should be a second pass? We typically have 'label' outside of props